src/OSPSuite.Core/Domain/Services/ParameterIdentifications/TransferOptimizedParametersToSimulationsTask.cs

```diff
@@ -52,17 +53,23 @@ public ICommand TransferParametersFrom(ParameterIdentification parameterIdentifi
             if (identificationParameter == null)
                throw new OSPSuiteException(Error.IdentificationParameterCannotBeFound(optimizedParameter.Name));
 
-            identificationParameter.AllLinkedParameters.Each(linkedParameter => macroCommand.Add(updateParameterValue(identificationParameter, optimizedParameter, linkedParameter)));
+            macroCommand.AddRange(setOptimalParameterValueIn(identificationParameter, optimizedParameter.Value));
          }
 
+         macroCommand.AddRange(parameterIdentification.AllFixedIdentificationParameters.SelectMany(x => setOptimalParameterValueIn(x, x.StartValue)));
```

Comment:
Transferring optimized parameter to simulation: Fixed parameter should also be taken into account